skyportal/models/user_notification.py

```diff
@@ -21,9 +22,28 @@
 from .comment import Comment
 from .listing import Listing
 from .facility_transaction import FacilityTransaction
+from ..email_utils import send_email
+from twilio.rest import Client as TwilioClient
+import gcn
+from sqlalchemy import or_
+
+from skyportal.models import Shift, ShiftUser
 
 _, cfg = load_env()
 
+log = make_log('notifications')
+
+account_sid = cfg["twilio.sms_account_sid"]
+auth_token = cfg["twilio.sms_auth_token"]
+from_number = cfg["twilio.from_number"]
+client = None
+if account_sid and auth_token and from_number:
+    client = TwilioClient(account_sid, auth_token)
+
+email = False
+if cfg["email_service"] == "sendgrid" or cfg["email_service"] == "smtp":
```

Comment:
Should we have a separate flag in the config like enabling email? I.e. you can have it all setup fine but just turn it off with a email=false?

Reply:
Right now it's what the email service is for kinda, if you don't specify one (that's what is done by default) it won't use it, and otherwise, it will. But maybe an extra flag to just deactivate it without having to modify your email config could be a nice addition?

Reply:
Yeah that's what I am thinking!